src/data/equiv/basic.lean

```diff
@@ -336,6 +336,12 @@ lemma perm_congr_symm_apply {α β : Type*} (e : α ≃ β) (p : equiv.perm β)
 protected lemma image_eq_preimage {α β} (e : α ≃ β) (s : set α) : e '' s = e.symm ⁻¹' s :=
 set.ext $ assume x, set.mem_image_iff_of_inverse e.left_inv e.right_inv
 
+@[simp] lemma symm_preimage_eq_image {α β} (e : α ≃ β) (s : set α) : e.symm ⁻¹' s = e '' s :=
+by symmetry; exact equiv.image_eq_preimage e s
```

Comment:
@gebner, I think you added these lemmas - are you in favor of this as a simp lemma?

Reply:
Also, this golfs to
```suggestion
@[simp] lemma symm_preimage_eq_image {α β} (e : α ≃ β) (s : set α) : e.symm ⁻¹' s = e '' s :=
(e.image_eq_preimage s).symm
```

Reply:
There's potentially a reason to prefer `⁻¹'` over `''`.  It has a much nicer definitional reduction (even though it is much harder to type).  I think that's why I didn't make this a simp lemma.

If this becomes a simp lemma, then `(symm_image_eq_preimage _ _).symm` should also be a simp lemma.